serverless/messaging/api-cli/connect-aws-cli.mdx

```diff
@@ -17,13 +17,12 @@ The AWS-CLI is an open-source tool built on top of the AWS SDK for Python (Boto)
 
 This guide shows you how to install the AWS-CLI and configure it to connect to Scaleway Messaging and Queuing SQS or SNS.
 
-<Macro id="iam-requirements" />
+## Requirements
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have [activated SQS and/or SNS](/serverless/messaging/how-to/get-started/#how-to-activate-sqs-or-sns)
-  - You have [generated credentials](/serverless/messaging/how-to/create-credentials/) for SQS/SNS
-</Message>
+- Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+- A Scaleway account logged into the [console](https://console.scaleway.com)
```

Comment:
I wonder if we can just get rid of this requirement since it's the same for all of them? Although I guess it's nice to provide the link to where the actions will be carried out...not sure. Could it be merged with the IAM requirement?

If instead of "Requirements" the wording was "Before you start" maybe we could have something like

**Before you start**
"Log into the Scaleway console, and ensure you are in an Organization of which you are the Owner or bla bla"

But then it would be back to repeating words for the following requirements
"Ensure you have activated SQS and/or SNS
"Ensure you have valid credentials"

Or maybe a format like:

**Before you start**
Log into the Scaleway console, and ensure you are in an Organization of which you are the Owner or bla bla. Ensure that you have already:
- Activated SQS and/or SNS
- Generated valid credentials